app/src/main/java/com/github/h3lp3rs/h3lp/database/FireDatabase.kt

```diff
@@ -78,7 +78,12 @@ class FireDatabase(path: String) : Database {
      * @return Future of double
      */
     override fun getDouble(key: String): CompletableFuture<Double> {
-        return get(key)
+        //This Fix is due to an misconception in firebase:
+        //Storing 3.0 in fire base will automatically transform it in an long integer.
+        //This result when getting it as an type error since long cannot be direct cast as double
+        //This is fix getting the field as the super class number and then casting it with its function

```

Comment:
I would just add a space at the beginning of each line. I also fixed a few typos. Feel free to change it again :) 
```suggestion
        // This Fix is due to a misconception in firebase:
        // Storing 3.0 in firebase will automatically transform it into a long integer.
        // This causes a type error when getting it since long cannot be directly cast to double.
        // This is fixed by getting the field as the superclass number and then casting it with its function.

```